electrumsv/wallet.py

```diff
@@ -4324,23 +4484,28 @@ async def _consume_mapi_callback_messages_async(self, state: ServerConnectionSta
             if self._stopping or self._stopped:
                 return
 
-            # We can now process the next batch of messages.
-            message_entries = state.mapi_callback_response_queue.get_nowait()
-            self._logger.debug(f"Got mAPI callback messages: {message_entries}")
             if state.mapi_callback_response_queue.qsize() == 0:
                 state.mapi_callback_response_event.clear()
+                continue
```

Comment:
Because if the queue is empty then `state.mapi_callback_response_queue.get_nowait()` will raise and crash the task.
And if there's no further work in the queue for processing then it should immediately go back to waiting